src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryTable.cs

```diff
@@ -15,7 +15,105 @@
 namespace Microsoft.PowerFx.Functions
 {
     internal static partial class Library
-    {
+    {
+        public static async ValueTask<FormulaValue> Collect(EvalVisitor runner, EvalVisitorContext context, IRContext irContext, FormulaValue[] args)
```

Comment:
>Collect [](http://example.com/codeflow?start=52&length=7)

Curious, why not in LibraryMutation.cs with other Collect impls?

Reply:
This should be removed. Thanks for catching it.